frontend/src/pages/Stats.tsx

```diff
@@ -0,0 +1,249 @@
+        <div className="flex justify-center max-w-fit items-center m-1 font-medium py-1 px-2 bg-transparent rounded-full text-whi bg-gray-100 border ">
+            <div className="text-xs font-normal leading-none max-w-full flex-initial">Hello</div>
+        </div>
+          {
+            data?.map((tags) => {
+              return  <span className="py-2.2 px-3.6 text-xs rounded-1.8 inline-block whitespace-nowrap text-center bg-slate-650 text-slate-750 align-baseline font-bold uppercase leading-none">{tags['tagName']}</span>
+            })
+          }
+      </div>
+      </div>
+    </div>
+  );
+};
+
+const UserCard = ({data } : {data : User}) => {
+  return (
+    <div key={data.id} className="relative flex flex-col text-gray-700 shadow-md w-auto mr-4 h-14 rounded-xl bg-clip-border transform transition-transform duration-300 hover:scale-105 border-2 my-2 ">
+      <div role="button"
+           className="flex mx-2 items-center w-full leading-tight transition-all rounded-lg outline-none text-start hover:bg-blue-gray-50 hover:bg-opacity-80 hover:text-blue-gray-900 focus:bg-blue-gray-50 focus:bg-opacity-80 focus:text-blue-gray-900 active:bg-blue-gray-50 active:bg-opacity-80 active:text-blue-gray-900 ">
+        <div className="grid mr-4 place-items-center">
+        { data.profilePic ? 
+          <img alt="candice" src={data.profilePic}
+            className="relative inline-block h-9 w-9 !rounded-full  object-cover object-center" />
+          :
+          <CircleUser size={40}  /> 
+          
+          }
+        </div>
+        <div>
+          <h6
+            className="block font-sans text-base antialiased font-semibold leading-relaxed tracking-normal text-blue-gray-900">
+            {data.name}
+          </h6>
+          <p className="block font-sans text-sm antialiased font-normal leading-normal text-gray-700">
+            {data.email}
+          </p>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const Graph = ({data } : {data : any}) => {
+  return (
+    <div className="lg:w-2/3 sm:w-full mb-8 p-8 ">
+      <div className='flex  flex-col col-span-12 md:col-span-3 border shadow-sm text-main bg-main'>
+        <div className="text-left p-4 overflow-scroll">
+          <h4 className="text-lg font-medium pb-2 text-nowrap overflow-hidden text-ellipsis">Monthly Blog Count</h4>
+          <Line data={data} 
+            options={{
+              scales : {
+                y: {
+                  beginAtZero: true,
+                  ticks: {
+                    precision: 0, // Display whole numbers only
+                  },
+                }
+            },
+            plugins: {
+              title: {
+                display: true,
+                text: "Number of Blogs posted montly"
+              },
+
+              legend: {
+                display: false
+              }
+            }
+        }} />
+        </div>
+      </div>
+    </div>
+  )
+}
+
+
+export default Stats
```

Comment:
1. Code is not formatted properly. Please use linting to fix it.
2. Please create stats folders in component and put all auxiliary components inside it.
3. Please attach a video recording of your final stats page. (Light & Dark mode, desktop & Mobile)

Reply:
Will do